contracts/test/unit/PCVGuardian.sol

```diff
@@ -0,0 +1,491 @@
+pragma solidity ^0.8.4;
+
+import {Vm} from "../utils/Vm.sol";
+import {DSTest} from "../utils/DSTest.sol";
+import {getCore, getAddresses, FeiTestAddresses} from "../utils/Fixtures.sol";
+
+import {ICore} from "../../core/ICore.sol";
+import {MockERC20} from "../../mock/MockERC20.sol";
+import {MockPCVDepositV2} from "../../mock/MockPCVDepositV2.sol";
+import {PCVGuardian} from "../../pcv/PCVGuardian.sol";
+
+contract PCVGuardianTest is DSTest {
+    ICore private core;
+    PCVGuardian private pcvGuardian;
+    MockERC20 private token;
+    MockPCVDepositV2 private pcvDeposit1;
+    MockPCVDepositV2 private pcvDeposit2;
+    MockPCVDepositV2 private pcvDeposit3;
+
+    Vm public constant vm = Vm(HEVM_ADDRESS);
+    FeiTestAddresses public addresses = getAddresses();
+
+    function setUp() public {
+        // get core
+        core = getCore();
+
+        // initialize mock tokens & deposits
+        token = new MockERC20();
+        pcvDeposit1 = new MockPCVDepositV2(address(core), address(token), 0, 0);
+        pcvDeposit2 = new MockPCVDepositV2(address(core), address(token), 0, 0);
+        pcvDeposit3 = new MockPCVDepositV2(address(core), address(token), 0, 0);
+        token.mint(address(pcvDeposit1), 100e18);
+        pcvDeposit1.deposit();
+        payable(address(pcvDeposit1)).transfer(1 ether);
+
+        // initialize pcv guardian
+        address[] memory safeAddresses = new address[](2);
+        safeAddresses[0] = address(pcvDeposit1);
+        safeAddresses[1] = address(pcvDeposit2);
+        pcvGuardian = new PCVGuardian(address(core), safeAddresses);
+        // give roles to pcv guardian
+        vm.startPrank(addresses.governorAddress);
+        core.grantPCVController(address(pcvGuardian));
+        core.grantGuardian(address(pcvGuardian));
+        vm.stopPrank();
+
+        // labels
+        vm.label(address(core), "core");
+        vm.label(address(token), "token");
+        vm.label(address(pcvDeposit1), "pcvDeposit1");
+        vm.label(address(pcvDeposit2), "pcvDeposit2");
+        vm.label(address(pcvDeposit3), "pcvDeposit3");
+        vm.label(address(pcvGuardian), "pcvGuardian");
+    }
+
+    // should have no safe addresses upon deployment when deployed with no safe addresses
+    function testDeploy1() public {
```

Comment:
I would make this test name more instructive. Maybe `testDeployNoSafeAddresses` and then for the next test, `testDeployWithSafeAddresses`